docs/apt.md

```diff
@@ -0,0 +1,48 @@
+# apt BASH in the Command Line!
+
+## Summary 
+
+<p> What is the apt BASH command? </p>
```

Comment:
The <p> tags are used for HTML, but for this repository we are using Markdown, which doesn't use these tags so they should be deleted.